docs/fundamentals/dashboard/standalone-for-python.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Using Aspire Dashboard with Python Applications
+description: How to use the Aspire Dashboard in a Python application.
+ms.date: 08/27/2024
+ms.topic: tutorial
+ms.author: aapowell
+---
+
+# Tutorial: Using the Aspire Dashboard with Python Applications
+
+The [Aspire dashboard](overview.md) provides a great UI for viewing telemetry, and is available as a standalone container image that can be used with any OpenTelemetry-enabled app. In this article, you'll learn how to:
+
+> [!div class="checklist"]
+>
+> - Start the Aspire dashboard in standalone mode.
+> - Use the Aspire dashboard with a Python application.
+
+## Prerequisites
+
+To complete this tutorial, you need the following:
+
+- [Docker](https://docs.docker.com/get-docker/)
+  - You can use an alternative container runtime, but the commands in this article are for Docker.
+- [Python 3.9 or higher](https://www.python.org/downloads/) locally installed
+- A sample application
+
+## Sample application
+
+This quickstart can be completed using either Flask, Django, or FastAPI. A sample application in each framework is provided to help you follow along with this quickstart. Download or clone the sample application to your local workstation.
+
+### [Flask](#tab/flask)
+
+```Console
+git clone https://github.com/Azure-Samples/msdocs-python-flask-webapp-quickstart
+```
+
+### [FastAPI](#tab/fastapi)
+
+```Console
+git clone https://github.com/Azure-Samples/msdocs-python-fastapi-webapp-quickstart.git
+```
+
+---
+
+To run the application locally:
+
+### [Flask](#tab/flask)
+
+1. Go to the application folder:
+
+    ```Console
+    cd msdocs-python-flask-webapp-quickstart
+    ```
+
+1. Create a virtual environment for the app:
+
+    [!INCLUDE [Virtual environment setup](./includes/standalone/virtual-environment-setup.md)]
+
+1. Install the dependencies:
+
+    ```Console
+    pip install -r requirements.txt
+    ```
+
+1. Run the app:
+
+    ```Console
+    flask run
+    ```
+
+1. Browse to the sample application at `http://localhost:5000` in a web browser.
+
+    :::image type="content" source="./media/standalone/run-flask-app-localhost.png" alt-text="Screenshot of the Flask app running locally in a browser":::
+
+### [FastAPI](#tab/fastapi)
+
+1. Go to the application folder:
+
+    ```Console
+    cd msdocs-python-fastapi-webapp-quickstart
+    ```
+
+1. Create a virtual environment for the app:
+
+    [!INCLUDE [Virtual environment setup](./includes/standalone/virtual-environment-setup.md)]
+
+1. Install the dependencies:
+
+    ```Console
```

Comment:
```suggestion
    ```python
```

Reply:
`pip` is a CLI tool (like NuGet), which is why it's `Console` (or should be `console`).

```suggestion
    ```console
```